iothub/service/src/DigitalTwin/Authentication/IotServiceClientCredentials.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Common;
+using Microsoft.Azure.Devices.Extensions;
+using Microsoft.Rest;
+
+namespace Microsoft.Azure.Devices.Authentication
+{
+    internal abstract class IotServiceClientCredentials : ServiceClientCredentials
```

Comment:
This class, and others below, need class comments. It won't block GA, but we should get them in by EOW.

Reply:
Will create a task in the PBI

Reply:
Done.